### PR TITLE
General clean-up 3

### PR DESCRIPTION
**Pull request by sjaeckel**
Next (and hopefully last) general clean-up

### CODE REVIEW

File: src/misc/crypt/crypt.c [hunk 1]
```diff
@@ -329,6 +329,9 @@ const char *crypt_build_settings =
 #if defined(LTC_MKAT)
     "   Katja\n"
 #endif
+#if defined(LTC_FILE_READ_BUFSIZE)
```
Comment (karel-m) [inline src/misc/crypt/crypt.c]: typo? `LTC_PK_MAX_RETRIES`

Comment (sjaeckel) [reply src/misc/crypt/crypt.c]: yippie c&p

**Comment by sjaeckel** [comment]
looks like my changes in hashsum broke coverage.sh

**Comment by karel-m** [comment]
I'd put `LTC_PK_MAX_RETRIES` here:
```C
#if defined(LTC_MECC) || defined(LTC_MRSA) || defined(LTC_MDSA) || defined(LTC_MKAT)
   /* Include the MPI functionality?  (required by the PK algorithms) */
   #define LTC_MPI
   #ifndef LTC_PK_MAX_RETRIES
      /* iterations limit for retry-loops */
      #define LTC_PK_MAX_RETRIES  20
   #endif
#endif
```

**Comment by karel-m** [comment]
Do we care about warnings in `demos/openssl-enc.c`?

Try:
```
make V=1 CFLAGS="-Wall -Wextra -Wwrite-strings -Werror -DLTM_DESC -I../libtommath" EXTRALIBS=../libtommath/libtommath.a all
```

**Comment by karel-m** [comment]
The fix is easy:
```diff
diff --git a/demos/openssl-enc.c b/demos/openssl-enc.c
index 21121017..3aca04f4 100644
--- a/demos/openssl-enc.c
+++ b/demos/openssl-enc.c
@@ -89,7 +89,7 @@ union paddable {
  * Output:       <no return>
  * Side Effects: print messages and barf (does exit(3))
  */
-void barf(char *pname, char *err)
+void barf(const char *pname, const char *err)
 {
    printf("Usage: %s <enc|dec> infile outfile passphrase [salt]\n", pname);
    printf("\n");
```

**Comment by karel-m** [comment]
The `LTC_PK_MAX_RETRIES` is sort of unaligned:
```
PRNG:
   Yarrow (LTC_YARROW_AES=2)
   SPRNG
   RC4
   ChaCha20
   Fortuna (LTC_FORTUNA_POOLS=32, LTC_FORTUNA_WD=10)
   SOBER128

PK Crypto:
   RSA (with blinding and CRT hardening)
   DH
   ECC (with blinding)
   DSA
 LTC_PK_MAX_RETRIES=20
MPI (Math):
   LTC_MPI
   LTM_DESC
   LTC_MILLER_RABIN_REPS=35

Compiler:
   CYGWIN Detected.
   GCC compiler 5.4.0.
   x86-64 detected.
```

File: src/misc/crypt/crypt.c [hunk 2]
```diff
@@ -329,6 +329,9 @@ const char *crypt_build_settings =
 #if defined(LTC_MKAT)
     "   Katja\n"
 #endif
+#if defined(LTC_PK_MAX_RETRIES)
+    " " NAME_VALUE(LTC_PK_MAX_RETRIES) " "
```
Comment (karel-m) [inline src/misc/crypt/crypt.c]: maybe:
```
-    " " NAME_VALUE(LTC_PK_MAX_RETRIES) " "
+    "   " NAME_VALUE(LTC_PK_MAX_RETRIES) "\n"
```

**Pull request merged**